web_development_101/the_basics/introduction_to_web_development.md

```diff
@@ -50,17 +50,6 @@ These are some of the basic tools you will use regularly. You may not know what
 * **[Git](https://git-scm.com/)**
 * **[GitHub](https://github.com/)**
 
-### Motivation
-
-Learning to code is incredibly rewarding but can also be difficult and frustrating.
-The strongest assets you can have as a student are a desire to build, a problem-solving mind, and persistence in the face of setbacks.
```

Comment:
I've moved this text over to the next lesson, "Gearing Up". 

https://github.com/TheOdinProject/curriculum/pull/12574